samples/nrf9160/location/Kconfig

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "Location sample"
+
```

Comment:
Add "new style" onboard/external antenna configuration.

Reply:
@hiltunent Please check if this is still valid and do necessary actions

Reply:
@hiltunent @trantanen Yes, we do need GNSS antenna configuration for the sample. I'm working on the sample and can add it.